### ISSUE TITLE
The rest of the game metadata entries

### ISSUE THREAD

**Issue by OceanUwU** [RintaDev5792/FunnyOS#2]
from https://sdk.play.date/2.5.0/Inside%20Playdate.html#pdxinfo

- [ ] `card-highlighted/` - animation to be played when a card is shown
- [ ] `card-pressed.png` - displayed on A button down when a card is shown
- [ ] `icon-highlighted/` - animation to be played while an icon is hovered
- [ ] `launchImage.png` - image that displays while a game is loading
- [ ] `launchImages/` - animation that plays after a game is launched
- [ ] `launchSoundPath` - (i *assume* this isnt implemented already) - short audio file to be played as the game launch animation is taking place.

**Comment by RintaDev5792** [comment]
yup, I've got these on the to-do list, thanks for noting it here!

**Comment by RintaDev5792** [comment]
finished these. Will be up in 1.1.0 in a bit.
